misk/src/main/kotlin/misk/MiskModule.kt

```diff
@@ -13,6 +13,8 @@ import javax.inject.Singleton
 
 class MiskModule : AbstractModule() {
   override fun configure() {
+    binder().disableCircularProxies()
```

Comment:
nice

Reply:
maybe leave a comment on why misk is opinionated about circular proxies.